extensions/bitwarden/src/search.tsx

```diff
@@ -0,0 +1,191 @@
+import {
+  ActionPanel,
+  List,
+  Icon,
+  showToast,
+  ToastStyle,
+  PushAction,
+  Detail,
+  CopyToClipboardAction,
+  getPreferenceValues,
+} from "@raycast/api";
+import { Item, Folder } from "./types";
+import { useEffect, useState } from "react";
+import treeify from "treeify";
+import { filterNullishPropertiesFromObject, codeBlock, titleCase } from "./utils";
+import { useBitwarden as useSessionToken } from "./hooks";
+import { TroubleshootingGuide, UnlockForm } from "./components";
+import { existsSync } from "fs";
+import { BitwardenApi } from "./api";
+import { dirname } from "path";
+
+const { cliPath, clientId, clientSecret, fetchFavicons } = getPreferenceValues();
+process.env.PATH = dirname(cliPath)
+const bitwardenApi = new BitwardenApi(clientId, clientSecret);
+
+export default function ListCommand(): JSX.Element {
+  if (!existsSync(cliPath)) {
+    return <TroubleshootingGuide />;
+  }
+
+  const [sessionToken, setSessionToken] = useSessionToken(bitwardenApi);
+
+  if (sessionToken === null) return <UnlockForm setSessionToken={setSessionToken} bitwardenApi={bitwardenApi} />;
+
+  return <ItemList bitwardenApi={bitwardenApi} sessionToken={sessionToken}/>;
+}
+
+function ItemList(props: { bitwardenApi: BitwardenApi; sessionToken: string | undefined }) {
+  const { bitwardenApi, sessionToken } = props;
+  const [state, setState] = useState<{ folders: Folder[]; items: Item[] }>();
+
+  const folderMap: Record<string, Folder> = {};
+  for (const folder of state?.folders || []) {
+    if (folder.id) folderMap[folder.id] = folder;
```

Comment:
Please don't omit the `{ }` in conditionals.
```suggestion
    if (folder.id) { folderMap[folder.id] = folder; }
```

Reply:
This is absolutely _not_ subjective. It's a dangerous practice in any language that allows this syntax. Here's [some further reading](https://dwheeler.com/essays/apple-goto-fail.html) on a bug encountered because of this practice.